README.md

```diff
@@ -93,18 +98,19 @@ subcommand:
 
 ## Initialize skeleton APB application example
 ```
-apb init my_apb -p name=my_apb_param,type=string,default=foo_param,description="sample description" --async=optional --bindable --organization my_organization
+apb init my_apb -p name=my_apb_param,type=string,default=foo_param,description="sample description" --async=optional --bindable --org my_organization
 ```
 
 This gives us the following example apb.yaml 
 ```
+id: c170da42-cb2b-4285-acab-260c1d788fd0
 name: my_apb
 image: my_organization/my_apb
 description: This is a sample application generated by apb init
 bindable: True
 async: optional
-parameters:
+parameters: 
```

Comment:
@mdshuai I misread the indent on line 113. If it's generating an invalid yaml can you open and issue for it?